services/cards/src/reducers/CombinedReducers.tsx

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import Cards from './Cards';
 import Filters from './Filters';
+import Ui from './Ui';
```

Comment:
I don't think `Ui.tsx` was committed.... I'd also recommend changing this to e.g. `Layout` as "Ui" seems overly general.